docs/carrier.md

```diff
@@ -215,6 +215,34 @@ To ensure compatibility, set the switch manufacturer to `bandwidth.com`.
 [carriers-main]: /carrier/img/carriers-main.png "Carriers"
 [carrier]: /carrier/img/carrier.png "Carrier Main"
 
+## Headers from Customers to Providers
```

Comment:
Add which headers as this is for X headers only

Reply:
done